Examples/ObjectiveC/MultipleImagesExample.m

```diff
@@ -0,0 +1,52 @@
+
+#import "MultipleImagesExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleMultipleImages = @"MultipleImagesExample";
+
+@interface MultipleImagesExample () <MGLMapViewDelegate>
+
+@end
+
+@implementation MultipleImagesExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    // Create and add a map view.
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds styleURL:[MGLStyle outdoorsStyleURL]];
+    
+    // Center the map on Yosemite National Park, United States.
+    [mapView setCenterCoordinate:CLLocationCoordinate2DMake(37.761, -119.624) zoomLevel:10 animated:NO];
+    mapView.autoresizingMask = UIViewAutoresizingFlexibleHeight | UIViewAutoresizingFlexibleWidth;
+    mapView.delegate = self;
+    [self.view addSubview:mapView];
+}
+
+- (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
+    // Add icons from the U.S. National Parks Service to the map's style.
+    [style setImage:[UIImage imageNamed:@"nps-restrooms"] forName:@"restrooms"];
+    [style setImage:[UIImage imageNamed:@"nps-trailhead"] forName:@"trailhead"];
+    [style setImage:[UIImage imageNamed:@"nps-picnic-area"] forName:@"picnic-area"];
+    
+    NSURL *url = [[NSURL alloc] initWithString:@"mapbox://examples.ciuz0vpc"];
```

Comment:
We might add some explanation of what this source is and how someone could create their own.

Minor point, but the `url` variable name could stand to be a little more descriptive, too.